data/scenarios/Tutorials/backstory.yaml

```diff
@@ -0,0 +1,46 @@
+name: Backstory
+description: |
+  Introduction to the backstory of Swarm.
+objectives:
+  - goal:
+      - In a shockingly original turn of events, you have crash landed
+        on an alien planet!  It's not clear how you'll ever get home,
+        but since you're here, you might as well explore a bit. Your
+        sensors indicate that the atmosphere is highly toxic, so
+        you'll have to stay inside your robotic base, with its
+        built-in life support system. However, you are stocked with
+        all the materials you need to build robots to explore
+        for you!
+      - To start, you only have some very basic
+        devices which can give your robots abilities like moving, turning,
+        grabbing things, and interpreting very simple imperative
+        programs. As you use your robots to gather resources, you will
+        be able to construct better devices, which in turn allow you to
+        construct robots with upgraded abilities and programming
+        language features, which in turn allow you to program more
+        sophisticated robots which in turn will... you get the
+        idea.
+      - To prepare you, this simulator will walk you through a series of hands-on
+        exercises that introduce you to the way robots work and the programming
+        language you will use to control them.
+      - When you're ready for your first challenge, close this dialog with Esc or Ctrl-G;
+        after a few seconds you will progress to the first challenge.
```

Comment:
How about `place "Ready"` or `say "Ready!"` once #565 is merged?

The first thing I did was close the message by mistake and then I was left in an empty world.

It would feel a bit more optimistic to start in world 0 with an entity that reminds you about <kbd>Ctrl</kbd><kbd>G</kbd>. :slightly_smiling_face:

Reply:
Yeah, I thought `place "Ready"` was strange when they have no idea what `place` means.  I like the idea of `say "Ready!"`.  But you're right that there should be an entity with some description text to remind you of stuff.  I'll add that.